src/main/java/dave/parser/Parser.java

```diff
@@ -0,0 +1,70 @@
+package dave.parser;
+
+import dave.command.*;
```

Comment:
Perhaps, you may want to consider explicitly listing out all the libraries imported